.github/workflows/codeql-analysis.yml

```diff
@@ -54,6 +54,16 @@ jobs:
       run: |
         xcodebuild -scheme FramesTests -destination "platform=iOS Simulator,name=iPhone 14 Pro,OS=latest" test
 
+    - name: Build iOS Example Frame
+      run: |
+        cd iOS\ Example\ Frame\ SPM
+        xcodebuild build -scheme  iOS\ Example\ Frame -destination "platform=iOS Simulator,name=iPhone 14 Pro,OS=latest"
+
+    - name: Build UITest
+      run: |
+        cd iOS\ Example\ Frame\ SPM
+        xcodebuild -scheme UITest -destination "platform=iOS Simulator,name=iPhone 14 Pro,OS=latest" test
```

Comment:
This will probably run all the UITests and will take 10 minutes. We might consider not including it maybe? or, we can add a build capability to the scheme

Reply:
Umm, yes, it is taking a lot of time to build. Better to remove it?

Reply:
can we just compile that target instead?

Reply:
Can you help with an example?

Reply:
I thought of selecting the `UITest` scheme on Xcode, then ticking the `Run` checkbox for the UITest scheme. After that, it should become a target that's runnable as well as testable. So that we can remove the `test` wording from the end inside the `yml` file

Reply:
actually, even without doing anything, just removing the `test` should do the trick

Reply:
@okhan-okbay-cko I had added Regression Tests but they are taking more than 15 mins to build alone. So, removed them. Do you suggest to add them?

Reply:
that's a bit unexpected. Which `xcodebuild` command was used please?

Reply:
I remember using:
xcodebuild -scheme Regression\ Tests -destination "platform=iOS Simulator,name=iPhone 14 Pro,OS=latest" test

Reply:
I believe we can remove `test` command from the end, after making that scheme runnable. Then it shouldn't take that long

Reply:
Without test command, I am getting below error, Do you have any workaround for this?

<img width="981" alt="Screenshot 2024-02-19 at 11 43 47 am" src="https://github.com/checkout/frames-ios/assets/156112796/6c2fc513-3462-4ea4-a342-d8522ab52a5e">